common/services/prismic/opening-times.ts

```diff
@@ -257,12 +257,9 @@ export function createExceptionalOpeningHoursDays(
     const type = period.type || 'other';
     const days = sortedDates
       .map(date => {
-        const matchingVenueGroup = groupedExceptionalDays.find(group =>
-          group.find(day => isSameDay(day.overrideDate, date, 'UTC'))
-        );
-        const matchingDay = matchingVenueGroup?.find(day =>
-          isSameDay(day.overrideDate, date, 'UTC')
-        );
+        const matchingDay = groupedExceptionalDays
+          .flat()
```

Comment:
Does this code run on the frontend, and if so does our polyfill.io config cover this method? https://caniuse.com/array-flat

Reply:
Yes, and yes.

https://github.com/wellcomecollection/wellcomecollection.org/blob/51765422723d8431e3f550f133503376f0dc73d2/common/views/components/PageLayout/PageLayout.tsx#L119